docs/developer/HootenannyConflatingANewFeatureTypeWithGenericConflation.asciidoc

```diff
@@ -385,12 +392,11 @@ a classification model and which are not.  There are two ways to come up with an
 
 One quick way to come up with an attribute set is:
 1. From the Preprocess tab, click the Choose button in the Filter section.
-2. Select Filter --> supervised --> attribute --> AttributeSelection
+2. Select Filter -> supervised -> attribute -> AttributeSelection
```

Comment:
The double line and gt resulted in a line followed by an arrow, which was ugly.  The single dash + gt left just an arrow and looks a lot cleaner.